docs/ADDRESS_LOOKUP.md

```diff
@@ -11,7 +11,63 @@
 - If you're integrating with Drop-in:
     - Implement the mandatory `onAddressLookupQueryChanged(query: String)` callback and optional `onAddressLookupCompletion(lookupAddress: LookupAddress)` callback.
```

Comment:
Suggestion as discussed earlier: consider removing these line and keeping the code sample only to make the page less crowded